charts/trino/values.yaml

```diff
@@ -62,17 +62,17 @@ server:
   exchangeManager: {}
   # server.exchangeManager -- Mandatory [exchange manager
   # configuration](https://trino.io/docs/current/admin/fault-tolerant-execution.html#id1).
+  # Used to set the name and location(s) of the spooling storage destination. To enable fault-tolerant execution,
```

Comment:
These are my changes I made on master, they shouldn't show up in the diff - a bad rebase?

Reply:
Sorry about that, I will fix it.

Reply:
squashing commits at 1AM, not a great idea...